src/main/java/com/enderio/base/mixin/ItemStackMixin.java

```diff
@@ -0,0 +1,23 @@
+package com.enderio.base.mixin;
+
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+import net.minecraftforge.common.extensions.IForgeItemStack;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+
+@Mixin(ItemStack.class)
+public abstract class ItemStackMixin implements IForgeItemStack {
+    @Shadow()
+    public abstract Item getItem();
+
+    @Override
```

Comment:
Don't override, inject instead. No one should make a new itemstack class, but it is still better imo.

Reply:
Ah good idea, ill get that tomorrow

Reply:
It might not be possible because it's a default method. An override would be good instead.